src/app/views/data/data.js

```diff
@@ -30,6 +54,28 @@ class DataViewController extends ADBMapController {
   _addGeoJSONLayer(geojson) {
     L.geoJson(geojson).addTo(this.map);
   }
+
+  _onResize() {
+    this.charts.width = document.querySelectorAll(".histogram .panel-content p")[0].clientWidth;
+    try {
```

Comment:
I'm assuming you have this wrapped in a try/catch because sometimes this throws an exception because you're already inside a digest cycle. Consider running line 59 in a `$timeout` call instead of this.

Typically the angular `$timeout` service can be thought of as a 'safe' `$apply` call. It's a wrapper around window.timeout, which always calls `$rootScope.$apply()` after the callback function runs.